core/modules/user/user.theme.inc

```diff
@@ -386,3 +386,68 @@ function template_preprocess_user_profile(&$variables) {
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid;
   $variables['theme_hook_suggestions'][] = 'user_profile__' . $account->uid . '__' . $view_mode;
 }
+
+/**
+ * Preprocess variables for user-simplified-page.tpl.php.
+ *
+ * @param array $variables
+ *   An associative array containing:
+ *   - form: The form structure to be rendered.
+ *   - links: A render array containing the links to other related forms.
+ *
+ * @see user-simplified-page.tpl.php
+ */
+function template_preprocess_user_simplified_page(&$variables) {
+  $form = $variables['form'];
+  if ($form['#form_id'] == 'user_login') {
+    $page_title = t('Login');
+  }
+  elseif ($form['#form_id'] == 'user_pass') {
+    $page_title = t('Reset password');
+  }
+  elseif ($form['#form_id'] == 'user_register_form') {
+    $page_title = t('Create new account');
+  }
```

Comment:
Is there some reason why `backdrop_get_title()` can't be used here? This is probably going to frustrate someone that the page title is getting overridden so late in the page building process.

```suggestion
  $page_title = backdrop_get_title();
```

(untested)

Reply:
It works!